client/src/util/typedCrossfilter/crossfilter.js

```diff
@@ -60,6 +60,8 @@ export default class ImmutableTypedCrossfilter {
   }
 
   setData(data) {
+    if (this.data === data) return this;
+    // console.log("...crossfilter set data, will drop cache");
```

Comment:
dead code!

Reply:
not dead - important for current perf work.  There are a very small number of these in the code base that I'd prefer to leave unless there is a strong reason to drop.

Reply:
oh you mean the console.log statements are needed for a perf work ticket that will be worked on after this PR is merged? If so, definitely makes sense to keep around, but should we add a comment so other people don't accidentally remove this intentional commented out console.log in their PRs down the road? Thank you!